### ISSUE TITLE
Client can't change/edit recurring donation

### ISSUE THREAD

**Issue by microsyskl** [iATSPayments/com.iatspayments.civicrm#192]
Test below is from a confirmation email client gets after setting up a recurring donation. It provides a link for them to edit/change/delete this recurring donation but it just spits out an XML coded page.

Donors aren't happy because they can't change their recurring donation.

I also have no means of going in and changing it either.

Currently using CiviCRM 4.6.13, iats 1.5.0

Test-drive Email / Receipt
This is a test-drive email. No live financial transaction has occurred.
Thank You for your gift to Operation Exodus USA
Please print this confirmation for your records.
Contribution Information
Amount $ 1.00
Date March 22nd, 2017 12:38 PM
Transaction # A99EA286:1490200737
This is a recurring contribution. You can cancel future contributions by visiting this web page.
https://www.iatspayments.com/NetGate/ProcessLink.asmx?WSDL

You can update recurring contribution amount or change the number of installments for this recurring contribution by visiting this web page.
https://www.iatspayments.com/NetGate/ProcessLink.asmx?WSDL

**Comment by adixon** [comment]
Yes, that's a known/old issue, related to core, I believe it's been fixed in more recent versions.
You can work around it by editing your templates and removing that line.
In any case, you should absolutely update your extension and your civicrm core both have security issues.

**Comment by microsyskl** [comment]
Thank you. In the meantime, How can I edit the recurring donation manually until I update the core and iATS extension?

**Comment by adixon** [comment]
You as an administrator can edit the credit card details using this:
https://github.com/iATSPayments/com.iatspayments.civicrm/wiki/Card-on-File
You modify the other aspects of a recurring contribution by following the edit link beside the information about the recurring contribution at the bottom of the contact's contribution tab (i.e., it's civicirm core functionality).
